package center.unit.beggar.exception;

public class ChallengeNotFoundException extends RuntimeException {
}
